resources/metadata/mtaBuild.yaml

```diff
@@ -127,14 +127,14 @@ spec:
             fields:
               - name: mtarFilePath
   containers:
-    - image: devxci/mbtci
+    - image: devxci/mbtci:1.0.14
```

Comment:
can users override this via project config? for example if your project requires java 8, you can still downgrade, right?

Reply:
yes. This is just the default.